src/main/java/com/exactprosystems/remotehand/web/actions/ExecuteJS.java

```diff
@@ -23,66 +30,72 @@
 import com.exactprosystems.remotehand.ScriptExecuteException;
 import com.exactprosystems.remotehand.web.WebAction;
 
-import static com.exactprosystems.remotehand.web.WebJsUtils.executeJsCommands;
-
 public class ExecuteJS extends WebAction
 {
 	private static final Logger logger = LoggerFactory.getLogger(ExecuteJS.class);
-	private static final String JS_COMMANDS = "commands";
-	private static final String NOTHING_TO_EXECUTE_MESSAGE = "Nothing to execute";
+	protected static final String JS_COMMANDS = "commands";
+	protected static final String NOTHING_TO_EXECUTE_MESSAGE = "Nothing to execute";
 	protected static final String DEFAULT_DELIMITER = ";";
 
-
 	@Override
-	public boolean isNeedLocator()
+	public String run(WebDriver webDriver, By webLocator, Map<String, String> params) throws ScriptExecuteException
 	{
-		return false;
+		executeJsCommands(webDriver, splitCommands(getJsScript(params)), getJsArguments(webDriver, webLocator, params));
+		return null;
 	}
 
-	@Override
-	public boolean isCanWait()
+	protected List<String> splitCommands(String jsScript) throws ScriptExecuteException
 	{
-		return false;
+		String[] splitScripts = jsScript.split(DEFAULT_DELIMITER);
+
+		List<String> commands = new ArrayList<>();
+		for (String splitScript : splitScripts)
+		{
+			if (splitScript.isEmpty())
+				continue;
+			commands.add(splitScript + DEFAULT_DELIMITER);
+		}
+
+		if (commands.isEmpty())
+			throw new ScriptExecuteException(NOTHING_TO_EXECUTE_MESSAGE);
+		return commands;
 	}
 
-	@Override
-	public String run(WebDriver webDriver, By webLocator, Map<String, String> params) throws ScriptExecuteException
+	protected String getJsScript(Map<String, String> params) throws ScriptExecuteException
 	{
-		List<String> commands = getCommands(params);
-		if (commands.isEmpty())
+		String jsScript = params.get(JS_COMMANDS);
+		if (StringUtils.isEmpty(jsScript))
 			throw new ScriptExecuteException(NOTHING_TO_EXECUTE_MESSAGE);
+		return jsScript;
+	}
 
-		executeJsCommands(webDriver, commands);
+	protected void executeJsCommands(WebDriver webDriver, List<String> commands, Object... args)
+			throws ScriptExecuteException
+	{
+		WebJsUtils.executeJsCommands(webDriver, commands, args);
+	}
 
+	protected Object[] getJsArguments(WebDriver webDriver, By webLocator, Map<String, String> params)
+	{
 		return null;
```

Comment:
NPE alert. This value passes into RemoteWebDriver.executeScript, where is not expected null values as varargs. Use new Object[0]